src/main/kotlin/by/architecture_map/belarus/service/ConstructionImageService.kt

```diff
@@ -0,0 +1,14 @@
+package by.architecture_map.belarus.service
+
+import by.architecture_map.belarus.entity.ConstructionImage
+
+interface ConstructionImageService {
+
+    fun create(image: ConstructionImage): ConstructionImage
+    fun getRandomAndUniqueImages(gotImagesId: String?): MutableList<ConstructionImage>
+    fun getByConstructionArchitecturalStyleId(architecturalStyleId: Int): MutableList<ConstructionImage>
```

Comment:
get what? ) in the method name we do not care about the parameters, please check details here https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

also, please check the Java Code Conventions and Kotlin Code Conventions